ONSdigital/dp-dd-frontend-controller
# feature/codedeploy

### What

* Add deployment scripts
* Add `bundle` and `deploy` stages to build pipeline

### How to review

* Check the build pipeline for this branch, it should be passing
* Check the CodeDeploy console, deployments for this app should have succeeded
* Check the configuration of the application is correct

Or ask for a walkthrough.

### Who can review

Anyone but myself.
